bin/test_integration_rhel

```diff
@@ -1,23 +1,57 @@
 #!/bin/bash -ex
 
+function print_help() {
+  cat << EOF
+
+Run integration tests on RHEL container
+
+Usage: test_integration_rhel [options]
+
+    -h, --help            Shows this help message.
+
+    --rhel-version=7  Run in RHEL 7 container.
+
+    --rhel-version=8  Run in RHEL 8 container.
+
+    -d, --debug           Step into the container.
+
+EOF
+}
+
+while true ; do
+  case "$1" in
+    #--rhel-version=* ) RHEL_VERSION="${option#*=}" ; shift ;;
```

Comment:
can you explain please how come [this](https://github.com/conjurinc/dap-package/blob/3a980a45792aac85cd278ad3f520617b020154cd/lib/user_interactions.sh#L136) works but in your script it doesn't?

Reply:
Because I did:
 `case "$1" in` and in the link you've sent it's  `case "$@" in` and it does work.

Reply:
Changed